### PR TITLE
[zh-cn]tasks/security/cert-management/plugin-ca-cert

### PR DESCRIPTION
**Pull request by yulng**
Sync 1 file with en:
[zh-cn]tasks/security/cert-management/plugin-ca-cert
Thanks

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 1]
```diff
@@ -171,11 +171,34 @@ test: yes
...
     {{< /text >}}
 
-*  移除 Istio 组件：按照[卸载说明](/zh/docs/setup/getting-started/#uninstall)进行移除。
+*  删除身份验证策略从“foo”命名空间中：
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*  删除示例应用“sleep”和“httpbin”`：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md:186]:
```suggestion
*  删除示例应用 `sleep` 和 `httpbin`：
```

File: content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 2]
```diff
@@ -171,11 +171,34 @@ test: yes
     $ rm -rf certs
     {{< /text >}}
 
-*  删除私密 `cacerts`、`foo` 和 `istio-system` 命名空间：
+*  删除私密 `cacerts`：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md:174]:
```suggestion
*  删除 Secret `cacerts`：
```

File: content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 3]
```diff
@@ -171,11 +171,34 @@ test: yes
...
 
     {{< text bash >}}
     $ kubectl delete secret cacerts -n istio-system
-    $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*  移除 Istio 组件：按照[卸载说明](/zh/docs/setup/getting-started/#uninstall)进行移除。
+*  删除身份验证策略从“foo”命名空间中：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md:180]:
```suggestion
*  从 `foo` 命名空间中删除身份验证策略：
```

File: content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 4]
```diff
@@ -171,11 +171,34 @@ test: yes
...
+    {{< /text >}}
+
+*  删除示例应用“sleep”和“httpbin”`：
+
+    {{< text bash >}}
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+    {{< /text >}}
+
+*  从集群中卸载Istio：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md:193]:
```suggestion
*  从集群中卸载 Istio：
```

File: content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 5]
```diff
@@ -171,11 +171,34 @@ test: yes
...
+
+    {{< text bash >}}
+    $ istioctl uninstall --purge -y
+    {{< /text >}}
+
+*  从集群中删除命名空间“foo”和“istio-system”：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/cert-management/plugin-ca-cert/index.md:199]:
```suggestion
*  从集群中删除命名空间 `foo` 和 `istio-system`：
```

**Review by windsonsea — COMMENTED** [review]
Need to follow English style to clear the lint errors.

**Review by windsonsea — APPROVED** [review]
LGTM

**Pull request merged**